components/common/Nav2.tsx

```diff
@@ -7,13 +7,20 @@ import { SERVER_BASE_URL } from "../../lib/utils/constant";
 
 const { Header } = Layout;
 
-{/*async function isLoggedIn() {
-  const data = await axios.get(`${SERVER_BASE_URL}/user`);
+async function isLoggedIn() {
```

Comment:
You can verify if the user is logged in with this piece of code `const { data: currentUser } = useSWR("user", storage);`. Basically when the user is logged in, the user's info is set in localstorage and we can check if the user is logged in or not by checking the localstorage.